step : 0.0
typeSignal : "step"
squareAmp : 10.0
squarePeriod : 3.0
sinAmp : 10.0
sinPeriod : 3.0
